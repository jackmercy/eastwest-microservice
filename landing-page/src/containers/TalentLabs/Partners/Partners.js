import { memo } from 'react';
import { useIntl } from 'react-intl';

import { Content } from '../../components';
import { partners } from '../../HomePage/Partners/partners.configs';

import useStyles from './partners.styles';

const Partners = () => {
  const intl = useIntl();
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <Content className={classes.content}>
        <h2 className={classes.title} data-test="partners-title">
          {intl.formatMessage({ id: 'HOMEPAGE.PARTNERS.TITLE' })}
        </h2>
        <div className={classes.partners}>
          {partners.map((item) => (
            <div key={item.name} className={classes.partnerItem}>
              <img
                loading="lazy"
                alt={item.name}
                title={item.name}
                src={item.logo}
                className={classes.partnerlogo}
              />
            </div>
          ))}
        </div>
      </Content>
    </div>
  );
};

export default memo(Partners);
