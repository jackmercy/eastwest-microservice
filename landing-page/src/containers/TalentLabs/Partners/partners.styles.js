import { createUseStyles } from 'react-jss';

import { rem } from '../../../utils/jss';
import BREAKPOINTS from '../../../constants/BREAKPOINTS';
import COLORS from '../../../constants/COLORS';

export default createUseStyles({
  root: { backgroundColor: COLORS.bgGray01 },
  content: { paddingTop: rem(80), paddingBottom: rem(80) },
  title: {
    fontWeight: 900,
    fontSize: rem(56),
    lineHeight: rem(72),
    marginTop: 0,
    marginBottom: rem(80),
    textAlign: 'center',
    transition: 'font-size .5s ease, margin .5s ease',
  },
  partners: { display: 'flex', flexWrap: 'wrap' },
  partnerItem: {
    flex: '20% 0',
    maxWidth: '20%',
    marginBottom: rem(64),
    padding: [0, rem(8)],
    boxSizing: 'border-box',
    display: 'flex',
    alignItems: 'center',
  },
  partnerlogo: {
    flex: 'none',
    display: 'block',
    maxWidth: rem(160),
    maxHeight: rem(66),
    margin: [0, 'auto'],
  },

  [`@media only screen and (max-width: ${BREAKPOINTS.maxLg}px)`]: {
    partnerItem: { flex: '25% 0', maxWidth: '25%', marginBottom: rem(48) },
  },
  [`@media only screen and (max-width: ${BREAKPOINTS.maxMd}px)`]: {
    content: { paddingTop: rem(32), paddingBottom: rem(32) },
    partnerItem: {
      flex: 'calc(100% / 3) 0',
      maxWidth: 'calc(100% / 3)',
      marginBottom: rem(32),
    },
  },
  [`@media only screen and (max-width: ${BREAKPOINTS.maxSm}px)`]: {
    title: { fontSize: rem(32), lineHeight: rem(40), marginBottom: rem(48) },
    partnerItem: { flex: 'auto' },
    partnerlogo: { maxWidth: rem(90) },
  },
});
