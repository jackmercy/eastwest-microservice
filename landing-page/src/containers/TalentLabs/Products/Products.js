import { memo, useCallback, useState } from 'react';
import { useIntl } from 'react-intl';
import classNames from 'classnames';
import { useMediaQuery } from 'react-responsive';

import { Content } from '../../components';
import BREAKPOINTS from '../../../constants/BREAKPOINTS';

import Tooltip from './Tooltip';
import useStyles from './products.styles';

const Products = () => {
  const intl = useIntl();
  const classes = useStyles();
  const isMobile = useMediaQuery({
    query: `(max-width: ${BREAKPOINTS.maxMd}px)`,
  });
  const [tooltipState, setTooltipState] = useState({
    visible: false,
    activeContent: -1,
  });

  const handleMouseEnter = (activeContent) => {
    setTimeout(() => {
      if (!tooltipState.visible) {
        setTooltipState({ visible: true, activeContent });
      }
    }, 50);
  };

  const handleMouseLeave = () => {
    setTimeout(() => {
      if (tooltipState.visible) {
        setTooltipState({ visible: false, activeContent: -1 });
      }
    }, 50);
  };

  // eslint-disable-next-line react/prop-types
  const TooltipContent = ({ title, image }) => (
    <>
      <h2 className={classes.tooltipTitle}>{title}</h2>
      <img
        loading="lazy"
        alt={title}
        src={image}
        className={classes.tooltipImage}
      />
    </>
  );

  const renderTooltipContent = useCallback(() => {
    switch (tooltipState.activeContent) {
      case 1:
        return (
          <TooltipContent
            image="/static/images/talent-labs/progress-01.svg"
            title={intl.formatMessage({
              id: 'TALENT_LABS.PRODUCTS.MODAL.PROGRESS.01.TITLE',
            })}
          />
        );
      case 2:
        return (
          <TooltipContent
            image="/static/images/talent-labs/progress-02.svg"
            title={intl.formatMessage({
              id: 'TALENT_LABS.PRODUCTS.MODAL.PROGRESS.02.TITLE',
            })}
          />
        );
      case 3:
        return (
          <TooltipContent
            image="/static/images/talent-labs/progress-03.svg"
            title={intl.formatMessage({
              id: 'TALENT_LABS.PRODUCTS.MODAL.PROGRESS.03.TITLE',
            })}
          />
        );
      case 4:
        return (
          <TooltipContent
            image="/static/images/talent-labs/progress-04.svg"
            title={intl.formatMessage({
              id: 'TALENT_LABS.PRODUCTS.MODAL.PROGRESS.04.TITLE',
            })}
          />
        );
      default:
        return <span>empty</span>;
    }
  }, [classes, tooltipState, intl]);

  const TooltipPointers = () =>
    (!isMobile ? (
      <>
        <div
          tabIndex="-1"
          role="presentation"
          className={classNames(classes.step, classes.step1)}
          onMouseOver={() => {
            handleMouseEnter(1);
          }}
          onMouseOut={handleMouseLeave}
          onFocus={() => {}}
          onBlur={() => {}}
        />
        <div
          tabIndex="-1"
          role="presentation"
          className={classNames(classes.step, classes.step2)}
          onMouseOver={() => {
            handleMouseEnter(2);
          }}
          onMouseOut={handleMouseLeave}
          onFocus={() => {}}
          onBlur={() => {}}
        />
        <div
          tabIndex="-1"
          role="presentation"
          className={classNames(classes.step, classes.step3)}
          onMouseOver={() => {
            handleMouseEnter(3);
          }}
          onMouseOut={handleMouseLeave}
          onFocus={() => {}}
          onBlur={() => {}}
        />
        <div
          tabIndex="-1"
          role="presentation"
          className={classNames(classes.step, classes.step4)}
          onMouseOver={() => {
            handleMouseEnter(4);
          }}
          onMouseOut={handleMouseLeave}
          onFocus={() => {}}
          onBlur={() => {}}
        />
        <Tooltip
          className={classes.tooltip}
          visible={tooltipState.visible}
          offsetY={10}
        >
          {renderTooltipContent()}
        </Tooltip>
      </>
    ) : null);

  return (
    <Content className={classes.root}>
      <h2 className={classes.title} data-test="products-title">
        {intl.formatMessage({ id: 'TALENT_LABS.PRODUCTS.TITLE' })}
      </h2>
      <p className={classes.description} data-test="products-description">
        {intl.formatMessage({ id: 'TALENT_LABS.PRODUCTS.DESCRIPTION' })}
      </p>
      <img
        alt="products"
        loading="lazy"
        className={classes.productImage}
        src="/static/images/talent-labs/product.png"
        srcSet={[
          '/static/images/talent-labs/product.png 500w',
          '/static/images/talent-labs/product@2x.png 1500w',
          '/static/images/talent-labs/product@3x.png 2000w',
        ].join(', ')}
      />
      <h2
        className={classNames(classes.title, classes.howItWorkTitle)}
        data-test="products-how-it-work-title"
      >
        {intl.formatMessage({ id: 'TALENT_LABS.PRODUCTS.HOW_IT_WORK' })}
      </h2>
      <div onMouseLeave={handleMouseLeave} className={classes.productSteps}>
        <img
          alt="products"
          loading="lazy"
          className={classes.productStepsImage}
          src={
            !isMobile
              ? '/static/images/talent-labs/product-steps.svg'
              : '/static/images/talent-labs/product-steps-mobile.svg'
          }
        />
        <TooltipPointers />
      </div>
    </Content>
  );
};

export default memo(Products);
