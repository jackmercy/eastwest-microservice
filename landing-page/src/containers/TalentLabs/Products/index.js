import dynamic from 'next/dynamic';

export default dynamic(() => import('./Products'), { ssr: false });
