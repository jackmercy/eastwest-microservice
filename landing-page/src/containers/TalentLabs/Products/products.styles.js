import { createUseStyles } from 'react-jss';

import { rem } from '../../../utils/jss';
import COLORS from '../../../constants/COLORS';
import BREAKPOINTS from '../../../constants/BREAKPOINTS';

export default createUseStyles({
  root: { paddingTop: rem(64), paddingBottom: rem(154) },
  title: {
    textAlign: 'center',
    fontWeight: 900,
    fontSize: rem(56),
    lineHeight: rem(72),
    marginTop: 0,
    marginBottom: rem(24),
    transition: 'font-size .5s ease',
  },
  description: {
    textAlign: 'center',
    fontWeight: 600,
    fontSize: rem(24),
    lineHeight: rem(40),
    marginTop: 0,
    marginBottom: rem(64),
    transition: 'font-size .5s ease',
  },
  productImage: {
    display: 'block',
    width: '100%',
    maxWidth: rem(1138),
    margin: [0, 'auto', rem(64)],
  },
  howItWorkTitle: { transform: 'translateY(50%)', marginBottom: 0 },
  productSteps: { position: 'relative' },
  productStepsImage: { display: 'block', width: '100%' },
  step: {
    position: 'absolute',
    borderRadius: '50%',
    cursor: 'pointer',
    zIndex: 1,
    width: '11%',
    '&::after': {
      content: '""',
      display: 'block',
      borderRadius: 'inherit',
      width: '100%',
      paddingTop: '100%',
    },
  },
  step1: { top: '27.8%', left: '28.7%' },
  step2: { top: '27.8%', left: '47.85%' },
  step3: { top: '27.8%', left: '67.65%' },
  step4: { top: '27.8%', left: '87.3%' },
  tooltip: {
    width: '100%',
    maxWidth: rem(614),
    padding: rem(32),
    backgroundColor: COLORS.bgWhite,
    boxShadow: '2px 4px 25px rgba(0, 0, 0, 0.25)',
    borderRadius: rem(12),
  },
  tooltipTitle: {
    fontWeight: 900,
    fontSize: rem(24),
    lineHeight: rem(32),
    textAlign: 'center',
    marginTop: 0,
    marginBottom: rem(16),
  },
  tooltipImage: { width: '100%', display: 'block' },

  [`@media only screen and (max-width: ${BREAKPOINTS.maxMd}px)`]: {
    root: { paddingTop: rem(32), paddingBottom: rem(16) },
    title: { fontSize: rem(40), lineHeight: rem(48), marginBottom: rem(16) },
    description: { fontSize: rem(20), lineHeight: rem(24) },
    howItWorkTitle: { transform: 'translateY(0)', marginBottom: rem(32) },
    step: { width: '43%' },
    step1: { top: '25.05%', left: '27%' },
    step2: { top: '45.3%', left: '27%' },
    step3: { top: '65.7%', left: '27%' },
    step4: { top: '86.05%', left: '27%' },
  },
  [`@media only screen and (max-width: ${BREAKPOINTS.maxSm}px)`]: {
    title: { fontSize: rem(32), lineHeight: rem(40) },
    description: { fontSize: rem(16) },
    tooltip: { padding: rem(16) },
  },
});
