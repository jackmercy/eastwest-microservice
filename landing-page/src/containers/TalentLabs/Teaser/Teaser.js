import { memo } from 'react';
import { useIntl } from 'react-intl';

import { Logo } from '../../../components';

import useStyles from './teaser.styles';

const Teaser = () => {
  const intl = useIntl();
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <div className={classes.coverImage} data-test="teaser-cover-image" />
      <div className={classes.content}>
        <Logo
          type="talent-labs"
          className={classes.logo}
          dataTest="teaser-cover-talent-labs-logo"
        />
        <h2 className={classes.title} data-test="teaser-title">
          {intl.formatMessage({ id: 'TALENT_LABS.TEASER.TITLE' })}
        </h2>
        <p className={classes.description} data-test="teaser-description">
          {intl.formatMessage({ id: 'TALENT_LABS.TEASER.DESCRIPTION' })}
        </p>
      </div>
    </div>
  );
};

export default memo(Teaser);
