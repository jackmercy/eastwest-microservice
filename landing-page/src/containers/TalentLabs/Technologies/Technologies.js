import { memo, useMemo } from 'react';
import { useIntl } from 'react-intl';

import { Content } from '../../components';

import useStyles from './technologies.styles';

const Technologies = () => {
  const intl = useIntl();
  const classes = useStyles();

  const configs = useMemo(
    () => [
      {
        label: intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.01.LABEL' }),
        title: intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.01.TITLE' }),
        description: intl.formatMessage({
          id: 'TALENT_LABS.TECHNOLOGIES.01.DESCRIPTION',
        }),
        src: '/static/images/talent-labs/people-01.png',
        srcSet: [
          '/static/images/talent-labs/people-01.png 500w',
          '/static/images/talent-labs/people-01@2x.png 2000w',
        ].join(', '),
        services: [
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.01.PROVIDE.01' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.01.PROVIDE.02' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.01.PROVIDE.03' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.01.PROVIDE.04' }),
        ],
      },
      {
        label: intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.02.LABEL' }),
        title: intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.02.TITLE' }),
        description: intl.formatMessage({
          id: 'TALENT_LABS.TECHNOLOGIES.02.DESCRIPTION',
        }),
        src: '/static/images/talent-labs/people-02.png',
        srcSet: [
          '/static/images/talent-labs/people-02.png 500w',
          '/static/images/talent-labs/people-02@2x.png 2000w',
        ].join(', '),
        services: [
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.02.PROVIDE.01' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.02.PROVIDE.02' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.02.PROVIDE.03' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.02.PROVIDE.04' }),
        ],
      },
      {
        label: intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.03.LABEL' }),
        title: intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.03.TITLE' }),
        description: intl.formatMessage({
          id: 'TALENT_LABS.TECHNOLOGIES.03.DESCRIPTION',
        }),
        src: '/static/images/talent-labs/people-03.png',
        srcSet: [
          '/static/images/talent-labs/people-03.png 500w',
          '/static/images/talent-labs/people-03@2x.png 2000w',
        ].join(', '),
        services: [
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.03.PROVIDE.01' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.03.PROVIDE.02' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.03.PROVIDE.03' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.03.PROVIDE.04' }),
        ],
      },
      {
        label: intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.04.LABEL' }),
        title: intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.04.TITLE' }),
        description: intl.formatMessage({
          id: 'TALENT_LABS.TECHNOLOGIES.04.DESCRIPTION',
        }),
        src: '/static/images/talent-labs/people-04.png',
        srcSet: [
          '/static/images/talent-labs/people-04.png 500w',
          '/static/images/talent-labs/people-04@2x.png 2000w',
        ].join(', '),
        services: [
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.04.PROVIDE.01' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.04.PROVIDE.02' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.04.PROVIDE.03' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.04.PROVIDE.04' }),
        ],
      },
      {
        label: intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.05.LABEL' }),
        title: intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.05.TITLE' }),
        description: intl.formatMessage({
          id: 'TALENT_LABS.TECHNOLOGIES.05.DESCRIPTION',
        }),
        src: '/static/images/talent-labs/people-05.png',
        srcSet: [
          '/static/images/talent-labs/people-05.png 500w',
          '/static/images/talent-labs/people-05@2x.png 2000w',
        ].join(', '),
        services: [
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.05.PROVIDE.01' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.05.PROVIDE.02' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.05.PROVIDE.03' }),
          intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.05.PROVIDE.04' }),
        ],
      },
    ],
    [intl],
  );

  return (
    <div className={classes.root}>
      <Content className={classes.content}>
        <h2 className={classes.title} data-test="technologies-title">
          {intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.TITLE' })}
        </h2>
        <p className={classes.description} data-test="technologies-description">
          {intl.formatMessage({ id: 'TALENT_LABS.TECHNOLOGIES.DESCRIPTION' })}
        </p>
        {configs.map((item) => (
          <div className={classes.group} key={item.label}>
            <div className={classes.groupIntroduce}>
              <p className={classes.techLabel}>{item.label}</p>
              <div className={classes.techInfo}>
                <img
                  loading="lazy"
                  className={classes.techImage}
                  alt={item.label}
                  src={item.src}
                  srcSet={item.srcSet}
                />
                <div className={classes.techSummary}>
                  <h2 className={classes.techTitle}>{item.title}</h2>
                  <p className={classes.techDescription}>{item.description}</p>
                </div>
              </div>
            </div>
            <div className={classes.groupServices}>
              <p className={classes.serviceLabel}>
                {intl.formatMessage({
                  id: 'TALENT_LABS.TECHNOLOGIES.PROVIDE.LABEL',
                })}
              </p>
              {item.services.map((service) => (
                <p className={classes.serviceItem} key={service}>
                  {service}
                </p>
              ))}
            </div>
          </div>
        ))}
      </Content>
    </div>
  );
};

export default memo(Technologies);
