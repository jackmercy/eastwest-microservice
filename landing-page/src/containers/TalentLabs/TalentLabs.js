import { memo } from 'react';

import { SEO } from '../../components';
import { DEFAULT_SEO } from '../../../configs';
import { Layout, Header, Footer } from '../components';
import CreateProject from '../HomePage/CreateProject';

import Teaser from './Teaser';
import Technologies from './Technologies';
import Products from './Products';
import Partners from './Partners';
import useStyles from './talentLabs.styles';

const TalentLabs = () => {
  const classes = useStyles();

  return (
    <Layout className={classes.layout}>
      <SEO {...DEFAULT_SEO.PAGES.TALENT_LABS} />
      <Header />
      <Teaser />
      <Technologies />
      <Products />
      <Partners />
      <CreateProject />
      <Footer />
    </Layout>
  );
};

export default memo(TalentLabs);
