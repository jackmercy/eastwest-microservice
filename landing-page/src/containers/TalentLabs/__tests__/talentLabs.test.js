import React from 'react';
import { mount, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import preloadAll from 'jest-next-dynamic';

import { Footer, Header } from '../../components';
import TalentLabs from '../TalentLabs';
import Teaser from '../Teaser';
import Technologies from '../Technologies';
import Products from '../Products/Products';
import Partners from '../Partners';
import CreateProject from '../../HomePage/CreateProject';

describe('Testing TalentLabs', () => {
  let wrapper;

  beforeAll(async () => {
    // Header and Footer included in AboutUs
    // And they are using dynamic import, see more in Header.js and Footer.js
    // So we need wait for it to load all before we can assert anything
    await preloadAll();
    wrapper = mount(<TalentLabs />);
  });

  it('should render match snapshot', () => {
    const wrapper = shallow(<TalentLabs />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should have a Header', () => {
    expect(wrapper.find(Header)).toHaveLength(1);
  });

  it('should have a Footer', () => {
    expect(wrapper.find(Footer)).toHaveLength(1);
  });

  it('should have a Teaser section', () => {
    expect(wrapper.find(Teaser)).toHaveLength(1);
  });

  it('should have a Technologies section', () => {
    expect(wrapper.find(Technologies)).toHaveLength(1);
  });

  it('should have a Products section', () => {
    expect(wrapper.find(Products)).toHaveLength(1);
  });

  it('should have a Partners section', () => {
    expect(wrapper.find(Partners)).toHaveLength(1);
  });

  it('should have a CreateProject section', () => {
    expect(wrapper.find(CreateProject)).toHaveLength(1);
  });
});
