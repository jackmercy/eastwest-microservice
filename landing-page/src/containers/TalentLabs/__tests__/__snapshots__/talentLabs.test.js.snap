// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing TalentLabs should render match snapshot 1`] = `
<Memo(Layout)
  className="layout-0-2-1"
>
  <SEO
    creator=""
    description="Our world-class team of agile engineers, product managers, and designers works alongside you to build and deploy products differently than you have before. In the process, you become enabled to do this work self-sufficiently in the future. Your team learns cutting-edge technology and best practices the best way possible: by doing it. The knowledge you absorb while working with Talent Labs is just as valuable as the finished product itself."
    image=""
    keywords=""
    siteName=""
    title="East West - Talent Labs"
    url="https://dev.eastwest.live/talent-labs"
  />
  <ForwardRef(LoadableComponent) />
  <Memo(Teaser) />
  <Memo(Technologies) />
  <ForwardRef(LoadableComponent) />
  <Memo(Partners) />
  <Memo(CreateProject) />
  <ForwardRef(LoadableComponent) />
</Memo(Layout)>
`;
